ocs_ci/framework/testlib.py

```diff
@@ -35,3 +36,32 @@ class EcosystemTest(BaseTest):
     Base class for E2E team
     """
     pass
+
+
+def skipif_ocs_version(expr_list):
+    """
+    This is a decorator to skip the test if condition evaluates to
+    true based on expression
+
+    Args:
+        expr_list (list): condition for which we need to check,
+            eg: ['<4.3', '>4.2'], ['<=4.3', '>=4.2']
```

Comment:
Would it make sense to handle expressions argument both as simple string and as a list? (e.g. if I want to specify only one condition  like `'>4.2'`, using `@skipif_ocs_version('>4.1')` is slightly more clear and easier than ` @skipif_ocs_version(['>4.1'])`.
And it might be more error prone, because with the current implementation if somebody will provide just string, it will iterate over each character of that string and might produce slightly confusing errors.

My suggestion is, to add following line at the beginning (and rename `expr_list` to `expressions`):
```python
expressions = [expressions] if isinstance(expressions, str) else expressions
```

Reply:
done